2018-11-26-sip-minutes.md

```diff
@@ -0,0 +1,159 @@
+
+*Better understanding* was enabled by in depth presentations and Q&As with the EPFL Dotty team; *the Approach* was agreed upon the first day which resulted in creating “FAQs about Scala 3” (see below) and *the Action Plan* was outlined and is still under construction; sveveral issues were opened and a project plan Meta programming in Scala 3 has been developed (please see the list at the end of this document).
+
+As the most important points and summary is reflected in “FAQs about Scala 3”, it will stand as an official “minutes” for this unique SIP meeting.
+
+The following document is not intended to fully answer the listed questions, rather to acknowledge them; the SIP Committee is outlining a frame and will pursue answers moving forward. Answers may change down the line, as the Process evolves. 
+
+## Frequently Asked Questions about Scala 3
+
+### What is the goal of Scala 3?
+This new iteration of the language will be focused on simplification, ergonomics, and on creating a stronger foundation for future evolution.
+In addition,  the goal is for Scala 3 to be friendlier to newcomers with a swift onboarding experience.
+As usual, existing Scala code should be able to transition to Scala 3 with minimum pain.
+
+
+### What’s the timeline for Scala 3?
+ Scala 3 is planned to be released in 2020. There will be two phases.
+The first phase will end in 2019 with a feature freeze. Before the feature freeze, the SIP Committee and the Scala 3 team will focus on ironing out the set of language features and concrete semantics.
+The second phase will continue into 2020, where the focus will shift to hardening, tooling and polishing documentation to ensure a smooth migration. 
+
+### Who’s behind Scala 3?
+Over the last five years, Prof. Martin Odersky and his team at EPFL have developed Scala 3. (Also, several Scala 2 features were first incubated in Scala 3.)
+Scala 3 will continue to leverage this collaboration with the Scala 2 team at Lightbend to improve the language for existing Scala users and ease migration. 
+
+
+### What’s the role of the SIP Committee in the Scala 3 migration?
+The SIP Committee takes account of the interests of the varied stakeholders of the Scala language. Regarding the transition to Scala 3, the Committee is tasked with:
+1. Curating language changes in the Scala specification;
+2. Providing recommendations for migration from Scala 2 to Scala 3;
+3. Work with the Scala Center to be a point of reference during the transition to Scala 3, incorporating feedback from implementation experience and from the community.
+
+During the meeting, the Committee came up with the curated list of the Core Scala 3 features, that are considered as a bundle that represents Scala 3, others are marked as "not core" or "essential". Please see the [full list here](https://docs.google.com/spreadsheets/d/1GWJUo0U3JbBtrfg5vqgb6H5S6wlU5HnTxebLcHwD1zw/edit?usp=sharing), naming the "core" features as follows:
```

Comment:
You can split this into two sentences.

```suggestion
During the meeting, the Committee came up with the curated list of the Core Scala 3 features, that are considered as a bundle that represents Scala 3. Other features are marked as "not core" or "essential". Please see the [full list here](https://docs.google.com/spreadsheets/d/1GWJUo0U3JbBtrfg5vqgb6H5S6wlU5HnTxebLcHwD1zw/edit?usp=sharing), naming the "core" features as follows:
```

What is the difference between "core" and "essential"?

Reply:
"Core" are well-defined features or changes that are already designed and implemented. We have now accepted them on principle, but the finer details are still up for discussion.

"Essential" are wide, *unsolved* areas for which the committee recognizes that we cannot decently ship Scala 3.0 without a final design and implementation to solve them. Nothing has been accepted yet because they're not even fully *designed* yet.

Reply:
Thanks. That might be useful information to include. (Unless it was somewhere else and I missed it...)

Reply:
Agreed, I'll add that part.
We had a public SIP meeting yesterday and we spoke about it more, so this part will be shown in the upcoming minutes too.
Thanks, @sjrd for the answer and (again) @hrhino for taking the time to go as detailed over this, I truly appreciate it!!!